kem/hybrid/hybrid.go

```diff
@@ -0,0 +1,292 @@
+// Package hybrid defines several hybrid classical/quantum KEMs.
```

Comment:
Is it possible to convert this package in a KEM concatenator, rather than listing specific kems inside of it
otherwise, lets rename package `pqhybrid`

Reply:
I don't think the user should be able to concatenate any KEMs themselves.  This package will list some of them.  I don't see why we have to reflect the fact that they're all PQ in the name.  And why is `pqhybrid` better than `classichybrid`? `classicpqhybrid` is too long.  Just `hybrid` is clear enough I think.